modules/jupyter/authentication/authenticator/setup.py

```diff
@@ -23,10 +23,10 @@
     long_description="long description here",
     long_description_content_type="text/markdown",
     url="https://github.com/chiayi/ai-on-gke/tree/main/ray-on-gke/user/jupyterhub/auth",
-    packages=['gcpiapjwtauthenticator'],
+    packages=setuptools.find_packages(),
     license='Apache 2.0',
     install_requires=[
-        "jupyterhub>=4.1.0",
+        "jupyterhub",
```

Comment:
I believe version 4.1.0 contains a fix for a known vulnerability and thats why we have `>=4.1.0"